src/test/TestUtils.cpp

```diff
@@ -194,6 +197,78 @@ genesis(int minute, int second)
         getTestDateTime(1, 7, 2014, 0, minute, second));
 }
 
+ConfigUpgradeSetKey
+upgradeSorobanNetworkConfig(std::function<void(SorobanNetworkConfig&)> modifyFn,
+                            SimulationPtr simulation)
+{
+    auto nodes = simulation->getNodes();
+    auto& lg = nodes[0]->getLoadGenerator();
+    auto& app = *nodes[0];
+
+    auto& complete =
+        app.getMetrics().NewMeter({"loadgen", "run", "complete"}, "run");
+    auto completeCount = complete.count();
+    // Only create an account if there are none aleady created.
+    if (app.getMetrics()
```

Comment:
Why do we need to create accounts here? We could just use the root account right?

Reply:
How can I use the root account? If I remove the account creation step I get: ` Account 0 must exist in the DB.`

Reply:
ah I see, loadgen doesn't mark root account as "available to use" for some reason. if it did, it would simplify things since you wouldn't need account creation + offset. You could modify this path https://github.com/stellar/stellar-core/blob/ad7b1646209fcd069c067a9a7730c9b6305be667/src/simulation/LoadGenerator.cpp#L646 

Instead, it would also return root account (mRoot) if it's available. That's lesser priority though, and can be done as a follow-up.

Reply:
can we track this issue somewhere though? It'd be nice to get rid of account creation, and manipulations with the offset, which are quite error-prone.

Reply:
https://github.com/stellar/stellar-core/issues/4445